src/App/Faq/Component/styles.css

```diff
@@ -0,0 +1,47 @@
+.wrapper-faq-component {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  width: 80%;
+  border: 1px solid black;
+  font-size: 18px;
+  color: #222222;
+  border-left: 5px solid #c8cbd1;
+  border-radius: 8px;
+  border-radius: 16px;
+  transition: 1s;
+}
+.faq-title {
+  display: flex;
+  left: 10px;
+  width: 95%;
+  align-items: center;
+  height: 20%;
+  border-bottom: 1px #ebebeb solid;
+}
+.faq-title p {
+  position: absolute;
+  top: 1px;
+  left: 10px;
+  font-weight: 700;
+  display: inline;
+  z-index: 3;
+  color: red;
+}
+.faq-title p:nth-of-type(2) {
+  margin-left: 20px;
+  color: #222222;
+}
+
+.faq-content {
+  position: absolute;
```

Comment:
Trochę nie rozumiem po co tutaj kombinowanie z `position: absolute` ? 😄

Reply:
...głównie dlatego, że takie zabawy z position absolute mogą skutkować rozjeżdżaniem się widoku na innych ekranach. Ale to omówimy sobie też jeszcze na zajęciach 😃